### PR TITLE
User authentication

### PR DESCRIPTION
**Pull request by VangSang0**
Please add some critques

### CODE REVIEW

File: templates/_layout.html
Comment (linalu12) [inline templates/_layout.html:1]: html looks good

File: app.py [hunk 1]
```diff
@@ -109,26 +109,33 @@ def signing_up():
     return redirect(url_for('sign_in'))
 
 
+@app.get('/logout')
+def logout():
 
+    session.pop('user_id', None)
+    return redirect(url_for('sign_in'))
```
Comment (linalu12) [inline app.py:116]: You could add a flash message saying "Successful Logout" or something along those lines.

**Comment by linalu12** [comment]
Added a comment, but otherwise everything looks great!!

File: templates/_layout.html
Comment (hasikom12345) [reply templates/_layout.html:1]: Agree, the layout looks good. I am searching for the option to approve the request

File: app.py
Comment (linalu12) [inline app.py:1]: Really nice validation for sign up and sign in form

**Review by linalu12 — APPROVED** [review]

**Pull request merged**